webapp/src/client/client.js

```diff
@@ -41,7 +42,7 @@ AuthenticationContext.prototype._addAdalFrame = function _addAdalFrame(iframeId)
         } else if (document.body && document.body.insertAdjacentHTML) {
             document.body.insertAdjacentHTML(
                 'beforeEnd',
-                '<iframe name="' + iframeId + '" id="' + iframeId + '" style="display:none"></iframe>'
+                '<iframe name="' + iframeId + '" id="' + iframeId + '" style="display:none"></iframe>',
```

Comment:
0/5 Since you're touching this code anyway

```suggestion
                `<iframe name="${iframeId}" id="${iframeId}" style="display:none"></iframe>`,
```